lxd/containers_post.go

```diff
@@ -13,7 +13,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/dustinkirkland/golang-petname"
+	petname "github.com/dustinkirkland/golang-petname"
```

Comment:
What's the reason behind that one?

Reply:
vim-go automatically runs `goimports` when saving a file. It formats the code and cleans up imports. That's what happened here ;)